packages/vue-composables/src/use-disclosure.ts

```diff
@@ -0,0 +1,95 @@
+import { computed, HTMLAttributes, ref, watchEffect } from "vue"
+import { useId } from "./use-id"
+
+export interface UseDisclosureProps {
+  isOpen?: boolean
+  defaultIsOpen?: boolean
+  onClose?(): void
+  onOpen?(): void
```

Comment:
Great work here, @TylerAPfledderer !

I've been thinking about the naming of the `onClose` and `onOpen` functions from the `useDisclosure` composable.

I was of the view that we should probably name them `open` and `close` because in Vue most event emitted in a `child -> parent` direction. So, the name of the function just needs to be `close` or `open`. That way the event would be something like `onClose`.

```vue
<!-- in app.vue template -->
<c-input @change="close" />
```

In the above example, the close function is not actually passed into the `<c-input />` component. Rather, a `change` event is emitted by the `<c-input />` to the `app.vue` component.

So it would make sense to name the function handler `close` instead of `onClose`. Hope this makes sense :)